tests/test_ptc.py

```diff
@@ -679,6 +679,45 @@ def ptcFitAndCheckPtc(
                     np.sqrt(self.noiseSq), localDataset.noise[ampName]
                 )
 
+    def test_lsstcam_samples(self):
+        for dense in [False, True]:
+            for mode in ["normal", "upturn", "dip"]:
+                rawMeans, rawVars, ptcTurnoff = self._getSampleMeanAndVar(dense=dense, mode=mode)
+
+                # We only need a single amp.
+                ampName = self.ampNames[0]
+                dataset = PhotonTransferCurveDataset([ampName], ptcFitType="EXPAPPROXIMATION")
+                dataset.rawExpTimes[ampName] = np.arange(len(rawMeans), dtype=np.float64) + 1.0
+                dataset.rawMeans[ampName] = rawMeans
+                dataset.rawVars[ampName] = rawVars
+                dataset.covariancesSqrtWeights[ampName] = np.repeat(
+                    np.ones((dataset.covMatrixSide, dataset.covMatrixSide)), len(rawMeans)
+                ).reshape((len(rawMeans), dataset.covMatrixSide, dataset.covMatrixSide))
+                dataset.covariancesSqrtWeights[ampName][:, 0, 0] = np.sqrt(rawVars)
+                mask = np.ones(len(rawMeans), dtype=np.bool_)
+                mask[~np.isfinite(rawMeans) | ~np.isfinite(rawVars)] = False
+                dataset.expIdMask[ampName] = mask
+
+                configSolve = copy.copy(self.defaultConfigSolve)
+                configSolve.doFitBootstrap = False
+                configSolve.doLegacyTurnoffSelection = False
+
+                solveTask = cpPipe.ptc.PhotonTransferCurveSolveTask(config=configSolve)
+
+                solvedDataset = solveTask.fitMeasurementsToModel(dataset)
+
+                # Check that the ptcTurnoff is what is expected.
+                self.assertFloatsAlmostEqual(solvedDataset.ptcTurnoff[ampName], ptcTurnoff)
```

Comment:
Can you add `msg` arguments to all of these asserts with the `dense` and `mode` parameters?  That should help identify problems when these tests fail in the future.